documentation/README.md

```diff
@@ -10,30 +10,20 @@ The documentation is built using [Docusaurus 2](https://docusaurus.io/). The dep
 ## Installation
 
 ```console
-npm i
-npm run setup
+yarn
 ```
 
-This command checks out a local copy of the wiki and creates links to the content.
+This command installs all needed dependencies.
```

Comment:
I'm getting the following error when trying to execute `yarn` locally, am I doing something wrong?

```
➤ YN0013: │ @iota-community/iota-wiki-cli@https://github.com/iota-community/iota-wiki-cli.git#commit=ef2f73d6464205353a1660413a667b6318f378a7 can't be found in the cache and will be fetched from the remote repository
➤ YN0018: │ @iota-community/iota-wiki-cli@https://github.com/iota-community/iota-wiki-cli.git#commit=ef2f73d6464205353a1660413a667b6318f378a7: The remote archive doesn't match the expected checksum
➤ YN0000: └ Completed in 1m 38s
➤ YN0000: Failed with errors in 1m 38s
```

Reply:
Hmm, thats strange. You are already the second person with that problem, but I can't reproduce. I will check again

Reply:
Probably just a Windows thing, possibly because git checks out the repo with CRLF line endings while the source uses LF, leading to a different checksum? Running `yarn` in WSL works fine.

For Windows one can work around this (locally) by adding `checksumBehavior: update` to the `.yarnc.yml` file, but that does change the checksum in the `yarn.lock` file. Just need to make sure not to commit that, since it would probably break the Wiki itself.

Reply:
Yeah after having 3 people under windows with the same problem we came to the conclusion that may be the problem.
Maybe we can find a nice solution for that. In the meantime just use https://gitpod.io/ :trollface: